src/ctl/ctl.c

```diff
@@ -244,21 +269,29 @@ static void ctl_query_cleanup_real_args(const struct ctl_node *n,
  */
 static int ctl_exec_query_read(void *ctx, const struct ctl_node *n,
                                enum ctl_query_source source, void *arg,
-                               struct ctl_index_utlist *indexes) {
+                               struct ctl_index_utlist *indexes,
```

Comment:
typedef ctl_query_source -> umf_ctl_query_source and remove "enum"

Reply:
Done